jwql/utils/utils.py

```diff
@@ -21,10 +21,8 @@
     https://gist.github.com/jhunkeler/f08783ca2da7bfd1f8e9ee1d207da5ff
 
     Various documentation related to JWST filename conventions:
-    - https://jwst-docs.stsci.edu/display/JDAT/File+Naming+Conventions+and+Data+Products
     - https://innerspace.stsci.edu/pages/viewpage.action?pageId=94092600
     - https://innerspace.stsci.edu/pages/viewpage.action?spaceKey=SCSB&title=JWST+Science+Data+Products
-    - https://jwst-docs.stsci.edu/display/JDAT/Understanding+Associations?q=association%20candidate
```

Comment:
Just noting here that these are being removed because the links no longer work.

Reply:
This might be out of the scope of this PR (if so, I can make this change in another), but I also noticed the View File Type 'i' on the image display pages was a broken link. It looks like it could be changed to the following though:
https://jwst-pipeline.readthedocs.io/en/latest/jwst/data_products/product_types.html

Reply:
Thanks @bsunnquist.  I will actually push this change to another branch I am working on (https://github.com/bourque/jwql/tree/bug-fixes-stakeholder-presentations) that has similar minor changes to the web app in response to the stakeholder presentations.